src/Umbraco.Web/Templates/TemplateUtilities.cs

```diff
@@ -74,16 +82,22 @@ public static string ParseInternalLinks(string text, UrlProvider urlProvider)
                 }
             }
 
+            if (stripUdiDataAttributes)
+            {
+                text = StripUdiDataAttributes(text);
+            }
+            
             return text;
         }
 
         /// <summary>
         /// Parses the string looking for the {localLink} syntax and updates them to their correct links.
         /// </summary>
         /// <param name="text"></param>
+        /// <param name="stripUdiDataAttributes"></param>
         /// <returns></returns>
         [Obsolete("Use the overload specifying all dependencies instead")]
-        public static string ParseInternalLinks(string text)
+        public static string ParseInternalLinks(string text,bool stripUdiDataAttributes = true)

```

Comment:
Adding new parameters to a obsolete method isn't ideal. Is it necessary?

Reply:
Yep without doing a lot of refactoring in the PVC this is needed. This is the method called by several PVC

Dave